docs/concepts/cluster-administration/networking.md

```diff
@@ -184,6 +184,12 @@ Follow the "With Linux Bridge devices" section of [this very nice
 tutorial](http://blog.oddbit.com/2014/08/11/four-ways-to-connect-a-docker/) from
 Lars Kellogg-Stedman.
 
+### Multus (a Multi Network plugin)
+
+[Multus](https://github.com/Intel-Corp/multus-cni) is a Multi CNI plugin to support the Multi Networking feature in Kubernetes using CRD based network objects in Kubernetes.
```

Comment:
Multi --> Multiple ?
Multi is OK though :)

Reply:
yes yes, Multi is good :)